relational/table.py

```diff
@@ -498,6 +503,31 @@ def _ami_info_mohcd(self, rows, proj):
                                     value=datum[1],
                                     data=datum[2]))
 
+    _IS_100_AFFORDABLE_FIELDMAP = {
+        'total_project_units': 'total_project_units',
+        'total_affordable_units': 'total_affordable_units',
+    }
+
+    def _is_100_affordable(self, rows, proj):
+        """Populates whether a project is 100% affordable, at least insofar
+        as we can tell from MOHCD data.
+        """
+        units = _get_mohcd_units(proj, MOHCDPipeline.NAME)
+        if units:
```

Comment:
Nit: Shouldn't we also check that units > 0? I guess I'm not sure if a 0 net units 0 bmr project will ever show up, but if it does i'm not sure we shoudl say it's 100 percent affordable?